src/util/MetadataController.ts

```diff
@@ -0,0 +1,84 @@
+import arrayifyStream from 'arrayify-stream';
+import { Readable } from 'stream';
+import { Stats } from 'fs';
+import streamifyArray from 'streamify-array';
+import { DataFactory, StreamParser, StreamWriter } from 'n3';
+import { LDP, RDF, STAT, TERMS, XML } from './Prefixes';
+import { NamedNode, Quad } from 'rdf-js';
+
+export const TYPE_PREDICATE = DataFactory.namedNode(`${RDF}type`);
+export const MODIFIED_PREDICATE = DataFactory.namedNode(`${TERMS}modified`);
+export const CONTAINS_PREDICATE = DataFactory.namedNode(`${LDP}contains`);
+export const MTIME_PREDICATE = DataFactory.namedNode(`${STAT}mtime`);
+export const SIZE_PREDICATE = DataFactory.namedNode(`${STAT}size`);
+
+export const CONTAINER_OBJECT = DataFactory.namedNode(`${LDP}Container`);
+export const BASIC_CONTAINER_OBJECT = DataFactory.namedNode(`${LDP}BasicContainer`);
+export const RESOURCE_OBJECT = DataFactory.namedNode(`${LDP}Resource`);
+export const DATETIME_OBJECT = DataFactory.namedNode(`${XML}dateTime`);
```

Comment:
Would move all these to util as well, but let's do that in https://github.com/solid/community-server/issues/57